packages/internal-test-helpers/lib/node-query.ts

```diff
@@ -26,23 +37,29 @@ export default class NodeQuery {
     Object.freeze(this);
   }
 
-  find(selector) {
+  find(selector: string | Element) {
     assertSingle(this);
 
     return this[0].querySelector(selector);
   }
 
-  findAll(selector) {
-    let nodes = [];
+  findAll(selector: string) {
+    let nodes: HTMLElement[] = [];
```

Comment:
I think this should be `Element` instead? And also should have a return type.

```suggestion
  findAll(selector: string): NodeQuery {
    let nodes: Element[] = [];
```